core/electron-manager/src/backend/ElectronAuthorizationRequestHandler.ts

```diff
@@ -15,10 +15,9 @@ import {
   AuthorizationResponseJson, AuthorizationServiceConfiguration, BasicQueryStringUtils,
 } from "@openid/appauth";
 import { NodeCrypto } from "@openid/appauth/built/node_support";
-import { BackendLoggerCategory } from "../BackendLoggerCategory";
 import { ElectronAuthorizationEvents } from "./ElectronAuthorizationEvents";
 
-const loggerCategory = BackendLoggerCategory.Authorization;
+const loggerCategory = "electron-backend";

```

Comment:
Do you want to add this to the enum so it can be configured more easily?

Reply:
no, this is internal-only code and logged only to local desktops.